google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -56,7 +57,6 @@
  *
  * @see <a href="https://cloud.google.com/storage/docs">Google Cloud Storage</a>
  */
-@InternalExtensionOnly
 public interface Storage extends Service<StorageOptions> {
```

Comment:
@JesseLovelace , @frankyn 

InternalExtensionOnly is necessary, otherwise this change will break API.

Reply:
Definitely missed this during review, thank you @dmitry-fa!